PyHa/annotation_post_processing.py

```diff
@@ -52,28 +59,32 @@ def annotation_chunker(kaleidoscope_df, chunk_length):
                 # Determining the end of a human label
                 maxval = int(
                     round(
-                        (species_df["OFFSET"][annotation] +
-                         species_df["DURATION"][annotation]) *
-                        1000,
-                        0))
+                        (
+                            species_df["OFFSET"][annotation]
+                            + species_df["DURATION"][annotation]
+                        )
+                        * 1000,
+                        0,
+                    )
+                )
                 # Placing the label relative to the clip
                 human_arr[minval:maxval] = 1
             # performing the chunk isolation technique on the human array
 
             for index in range(potential_annotation_count):
-                chunk_start = index * (chunk_length*1000)
-                chunk_end = min((index+1)*chunk_length*1000,arr_len)
-                chunk = human_arr[int(chunk_start):int(chunk_end)]
+                chunk_start = index * (chunk_length * 1000)
+                chunk_end = min((index + 1) * chunk_length * 1000, arr_len)
+                chunk = human_arr[int(chunk_start) : int(chunk_end)]
                 if max(chunk) >= 0.5:
-                    row = pd.DataFrame(index = [0])
+                    row = pd.DataFrame(index=[0])
                     annotation_start = chunk_start / 1000
-                    #updating the dictionary
+                    # updating the dictionary
                     row["IN FILE"] = clip
                     row["CLIP LENGTH"] = clip_len
                     row["OFFSET"] = annotation_start
                     row["DURATION"] = chunk_length
                     row["SAMPLE RATE"] = sr
                     row["MANUAL ID"] = bird
                     row["CHANNEL"] = 0
-                    output_df = pd.concat([output_df,row], ignore_index=True)
-    return output_df
\ No newline at end of file
+                    output_df = pd.concat([output_df, row], ignore_index=True)
```

Comment:
Truth be told if this is getting changed, what should happen instead is we save each row to a list, and concat at the end.  See https://pandas.pydata.org/docs/reference/api/pandas.concat.html

Not an issue for this pr, just something that came to mind.